src/components/Navbar.js

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import Nav from "react-bootstrap/Nav";
+import Navbar from "react-bootstrap/Navbar";
+import NavDropdown from "react-bootstrap/NavDropdown";
+import Link from "react-router-dom/Link";
+import Button from "react-bootstrap/Button";
+import "bootstrap/dist/css/bootstrap.min.css";
+import "../styles/Navbar.css";
+
+function MyNav(props) {
+  return (
+    <Navbar className="navbar" expand="md">
+      <Link to="/">
+        <Navbar.Brand className="brand">
+          <p>A N D R E S</p>
+        </Navbar.Brand>
+      </Link>
+      <Navbar.Toggle aria-controls="basic-navbar-nav" />
+      <Navbar.Collapse id="basic-navbar-nav">
+        <Nav className="mr-auto"></Nav>
+        <Nav>
+          <Navbar.Text className="nav-link">
+            <Link to="/">
+              <p>Home</p>
+            </Link>
+          </Navbar.Text>
+          <NavDropdown title="Locations" id="basic-nav-dropdown">
+            <NavDropdown.Item href="/restaurants/antarctic">Antarctic</NavDropdown.Item>
+            <NavDropdown.Item href="/restaurants/arctic">Arctic</NavDropdown.Item>
+            <NavDropdown.Item href="/restaurants/sahara">Sahara</NavDropdown.Item>
+            <NavDropdown.Item href="/restaurants/arabian">Arabian</NavDropdown.Item>
+            <NavDropdown.Item href="/restaurants/gobi">Gobi</NavDropdown.Item>
```

Comment:
file paths may require a period at the start - ./

Reply:
Adding a "." caused router link issues due to the nested routes. So this has not been changed.

Reply:
Using href was causing a page refresh so I am using LinkContainer from the newly added package react-router-bootstrap.